configator/generate_readme.py

```diff
@@ -0,0 +1,11 @@
+"""Element of the Configator tool to enable default README.md generation"""
+
+print("Generating default README...!")
+
+def create_readme():
+    """ method to generate README
+    """
+    with open("../README.md", "w", encoding="utf8") as read_me:
```

Comment:
This will create a `README.md` file that is outside of the user's current directory. Remember that this tool will be run ***inside*** the directory that the user wishes to generate files in. Additionally, this will overwrite any existing `README.md`, which is definitely not ideal. Lastly, the written content here is not complete.

Reply:
As a future user of this tool, I would say that the generation of the README file is the least important feature of this program. So, if there are time constraints, consider removing this feature altogether.